source/ProtonPack/ProtonPack.ino

```diff
@@ -4561,24 +4608,24 @@ void checkWand() {
 
               switch(i_mode_year) {
                 case 2021:
-                  stopEffect(S_AFTERLIFE_CROSS_THE_STREAMS_END);
-                  stopEffect(S_AFTERLIFE_CROSS_THE_STREAMS_START);
+                  //stopEffect(S_AFTERLIFE_CROSS_THE_STREAMS_END);
```

Comment:
Were these left commented out for testing, or to eventually be removed>

Reply:
At some point I'd like to re implement but because of the wand overload feature they are no longer critical. I was lessening the burden on the serial buffer queue. Particularly on sound effects which are very short in nature and can play out instead of sending a serial command to stop it.